cdk_framework/EKS/test/cluster-deployment.test.ts

```diff
@@ -0,0 +1,77 @@
+import * as cdk from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import { readFileSync, writeFileSync } from 'fs';
+import { validateClustersConfig } from '../lib/utils/validate';
+import { Stack, StackProps, aws_eks as eks, aws_ec2 as ec2} from 'aws-cdk-lib';
+import { ClusterStack } from '../lib/stacks/cluster-stack';
+import { VPCStack } from '../lib/utils/vpc-stack';
+const yaml = require('js-yaml')
+
+// example test. To run these tests, uncomment this file along with the
+// example resource in lib/my-project-stack.ts
+test('ClusterTest', () => {
+    var app = new cdk.App();
+    // WHEN
+    const route = process.env.CDK_CONFIG_PATH ||  __dirname + '/config/clusters.yml';
+    const raw = readFileSync(route)
+    const data = yaml.load(raw)
```

Comment:
We should be reading env variables here. Instead provide a test config that is used for every test run.

Reply:
Environment variables? Why would that make more sense than a configuration file? Should I make up values in the test file itself?